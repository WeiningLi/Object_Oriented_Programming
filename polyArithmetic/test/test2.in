create a 0/1 5 0/-1 2 0/1 1 0/2 0
print a 
create b 0/1 0
print b
create c 1/1 1
print c
create d 1/1 0
print d
subtract a b
subtract b c
subtract c d
subtract b d
add a b
add a c
add c a
add c d
add a d
divide a d
divide b c
move_assign d a
move_assign c b
print a
print b
quit
