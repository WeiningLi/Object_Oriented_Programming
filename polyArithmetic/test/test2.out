0
0
(1)x
(1)
0
(-1)x
(1)x + (-1)
(-1)
0
(1)x
(1)x
(1)x + (1)
(1)
Quotient: 0
Remainder: 0
Quotient: 0
Remainder: 0
(1)
(1)x
(1)
(1)x
